docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
       - write about start()
```

Comment:
This TODO comment can be removed.

Reply:
All TODOs will be removed in the last commit.